### PR TITLE
Update production deployment README

### PR DESCRIPTION
**Pull request by eatyourgreens**
Update the production deployment section to add some more info about our Docker images and where they are deployed.

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?
- [ ] Is the changelog updated?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && yarn bootstrap` and app works as expected?
- [ ] Can you run a [production build](https://github.com/zooniverse/front-end-monorepo#getting-started) of the app?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,7 +115,12 @@ Apps should have their directory names prefixed with `app-`, e.g. `/project` bec
 
 ## Production deployment
 
-Deploys to production are handled by [Jenkins](https://jenkins.zooniverse.org/job/Zooniverse%20GitHub/job/front-end-monorepo/). Firstly, a base Docker image is created which installs and builds the `lib-` packages, and that's used as a base image for creating `app-` images, which are then deployed to Kubernetes.
+Deploys to production and staging are handled by [Jenkins](https://jenkins.zooniverse.org/job/Zooniverse%20GitHub/job/front-end-monorepo/). Firstly, a base Docker image (`zooniverse/front-end-monorepo`) is built with the top-level Dockerfile, which installs and builds production versions of the `lib-` packages.
```
Comment (rogerhutchings) [inline README.md]:
```suggestion
Deploys to production and staging are handled by [Jenkins](https://jenkins.zooniverse.org/job/Zooniverse%20GitHub/job/front-end-monorepo/). Firstly, a base Docker image ([`zooniverse/front-end-monorepo`](https://hub.docker.com/r/zooniverse/front-end-monorepo)) is built with the top-level Dockerfile, which installs and builds production versions of the `lib-` packages. 
```

File: README.md [hunk 2]
```diff
@@ -115,7 +115,12 @@ Apps should have their directory names prefixed with `app-`, e.g. `/project` bec
 
 ## Production deployment
 
-Deploys to production are handled by [Jenkins](https://jenkins.zooniverse.org/job/Zooniverse%20GitHub/job/front-end-monorepo/). Firstly, a base Docker image is created which installs and builds the `lib-` packages, and that's used as a base image for creating `app-` images, which are then deployed to Kubernetes.
+Deploys to production and staging are handled by [Jenkins](https://jenkins.zooniverse.org/job/Zooniverse%20GitHub/job/front-end-monorepo/). Firstly, a base Docker image (`zooniverse/front-end-monorepo`) is built with the top-level Dockerfile, which installs and builds production versions of the `lib-` packages. 
+
+That image is used as a base image to build `zooniverse/fe-project` and `zooniverse/fe-content-pages` images, from the Dockerfiles in `packages/app-project` and `packages/app-content-pages`. Those images run production (static) builds of the NextJS apps and are then deployed to Kubernetes.
```
Comment (rogerhutchings) [inline README.md]:
```suggestion
That image is used as a base image to build [`zooniverse/fe-project`](https://hub.docker.com/r/zooniverse/fe-project) and [`zooniverse/fe-content-pages`](https://hub.docker.com/r/zooniverse/fe-content-pages) images, from the Dockerfiles in `packages/app-project` and `packages/app-content-pages`. Those images run production (static) builds of the NextJS apps and are then deployed to Kubernetes.
```

File: README.md (hunk 1 above)
Comment (rogerhutchings) [reply README.md]: Adds a link on Docker Hub

File: README.md (hunk 2 above)
Comment (rogerhutchings) [reply README.md]: Adds links to Docker Hub

File: README.md (hunk 1 above)
Comment (eatyourgreens) [reply README.md]: This text is pretty much all deleted in #1126.

File: README.md (hunk 2 above)
Comment (eatyourgreens) [reply README.md]: Same here.

**Comment by srallen** [comment]
My suggestions have merged and now can use a re-review.

**Comment by eatyourgreens** [comment]
Looks good to me but I can't approve my own PR.

**Pull request merged**